rules/php80/src/NodeAnalyzer/SwitchAnalyzer.php

```diff
@@ -12,17 +12,26 @@ final class SwitchAnalyzer
 {
     public function hasEachCaseBreak(Switch_ $switch): bool
     {
-        foreach ($switch->cases as $case) {
-            foreach ($case->stmts as $caseStmt) {
-                if (! $caseStmt instanceof Break_) {
-                    continue;
-                }
+        $totalCases = count($switch->cases);
+        if ($totalCases === 1) {
+            return false;
+        }
 
+        foreach ($switch->cases as $key => $case) {
+            if ($key === $totalCases - 1) {
                 return true;
             }
+
+            foreach ($case->stmts as $caseStmt) {
+                if ($caseStmt instanceof Break_) {
+                    continue 2;
+                }
+            }
```

Comment:
I'm not sure the behavior will be appropriate in this rule. However, `continue;` (without arguments) behaves the same as ' break;`

See https://github.com/rectorphp/rector/issues/5646